plugins/bazaar/src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -48,20 +49,33 @@ const useStyles = makeStyles({
     WebkitLineClamp: 7,
     WebkitBoxOrient: 'vertical',
     overflow: 'hidden',
+    color: 'transparent',
+    '-webkit-background-clip': 'text',
+  },
+  fullDescription: {
+    height: '10.2rem',
+    backgroundImage:
+      theme.palette.type === 'light'
```

Comment:
If you use theme.palette.navigation.background, you may not have to check if it is a light theme or not.

Reply:
I can't get it to work, could you maybe show me how you mean? :)